docs/components/navigator.md

```diff
@@ -0,0 +1,115 @@
+component's model data. The component stops reading these inputs when the last gamepad is disconnected.
+
+## Using this grade
+
+This grade can work without passing any option to it. An instance of the `navigator` does not require any options to be
+passed and can listen to the gamepad events and read the gamepad inputs. To use it, we can create an instance of the
+component or use it as a grade another component:
+
+``` javascript
+// Either create an instance of the navigator.
+var navigatorInstanceOne = gamepad.navigator();
+
+// Otherwise create a custom component using the navigator grade.
+fluid.defaults("my.navigator.grade", {
+    gradeNames: ["gamepad.navigator"]
+};
+var navigatorInstanceTwo = my.navigator.grade();
+```
+
+## Component Options
+
+The following component configuration options are supported:
+
+| Option | Type | Default Value | Description |
+| :---: | :---: | :---: | :--- |
+| `windowObject` | `{Object}` | `window` | The global window object from which the navigator derives various methods to read gamepad inputs and attach listeners to it for various gamepad events. |
+| `frequency` | `{Integer}` | 50 | The frequency (in ms) at which the navigator will read gamepad inputs. |
+
+These options can be provided to the component in the following ways:
+
+```javascript
+// Either pass the options inside the object as an argument while creating an instance of the navigator.
+var navigatorInstanceOne = gamepad.navigator({
+    windowObject: myCustomWindowObject,
+    frequency: 100
+});
+
+// Otherwise pass it as default options in a custom component using the navigator grade.
+fluid.defaults("my.navigator.grade", {
+    gradeNames: ["gamepad.navigator"],
+    windowObject: myCustomWindowObject,
+    frequency: 100
+};
+var navigatorInstanceTwo = my.navigator.grade();
+```
+
+Though the `windowObject` is a configurable option, it should have all the properties and methods that the navigator
+uses. Otherwise, the navigator won't work and will throw errors. You can also use a mutated `window` object and pass it
+into the component if you want to achieve a different behavior. That'll be less likely to throw errors and is the same
+pattern used in our tests.
+
+## Invokers
+
+### `{navigator}.attachListener(that)`
+
+- `that {Object}` The gamepad navigator component.
+- Returns: Nothing.
+
+It attaches event listeners to the `windowObject` for the `gamepadconnected` and `gamepaddisconnected` events. When
+these events are fired, the navigator fires its native `onGamepadConnected` or `onGamepadDisconnected` events,
+depending upon the former event. These events have listeners attached to it, which start reading the gamepad inputs or
+stop reading it further.
+
+### `{navigator}.onConnected(that)`
+
+- `that {Object}` The gamepad navigator component.
+- Returns: Nothing.
+
+It is the listener for the gamepad navigator component's event `onGamepadConnected`. When it is called, it initiates
```

Comment:
You can simplify all of these "it is the listener" phrases to simply say "Listens".